packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -44,21 +53,26 @@ import {
     FLAG_TYPE,
     INTEGER_TYPE,
     LOOKUP_TYPE,
+    ONTOLOGY_LOOKUP_TYPE,
     PARTICIPANT_TYPE,
-    PropDescType,
     PROP_DESC_TYPES,
+    PropDescType,
     READONLY_DESC_TYPES,
     SAMPLE_TYPE,
     TEXT_TYPE,
     USERS_TYPE,
-    ONTOLOGY_LOOKUP_TYPE,
 } from './PropDescType';
+import {removeFalseyObjKeys, reorderSummaryColumns} from "./propertiesUtil";
 
 export interface IFieldChange {
     id: string;
     value: any;
 }
 
+export interface DomainOnChange {
```

Comment:
From what I can tell this was only used in the `DomainRow.tsx` place where it was refactored out of. Does it still need to be pulled out into its own interface? Maybe I'm just missing a usage in my search.

Reply:
DomainOnChange is used in getGridColumns()'s onFieldsChange parameter, as well as the place you found in DomainRow. 
I pulled it out mainly because it's a long function type signature (and imo was making getGridColumns()'s params a little hard to read), so it's mostly a stylistic change, and I'd be fine with putting it back in!